applications/StructuralMechanicsApplication/tests/test_patch_test_cr_beam.py

```diff
@@ -19,7 +19,26 @@ def _add_dofs(self,mp):
         KratosMultiphysics.VariableUtils().AddDof(KratosMultiphysics.ROTATION_X, KratosMultiphysics.TORQUE_X,mp)
         KratosMultiphysics.VariableUtils().AddDof(KratosMultiphysics.ROTATION_Y, KratosMultiphysics.TORQUE_Y,mp)
         KratosMultiphysics.VariableUtils().AddDof(KratosMultiphysics.ROTATION_Z, KratosMultiphysics.TORQUE_Z,mp)
-    
+
+    def _add_expl_dofs(self,mp):
+        KratosMultiphysics.VariableUtils().AddDof(StructuralMechanicsApplication.MIDDLE_VELOCITY_X,mp)
```

Comment:
The middle velocity is a dof?

Reply:
actually it's not I got rid of that also in the explicit_dynamic_solver.py